### DISCUSSION TITLE
Support for Yarn Workspaces

### DISCUSSION THREAD

**Discussion by tinleym** [RyanClementsHax/storybook-addon-next#13] [Ideas]
This plugin (reasonably) expects to find the next package in the node modules of the project parent folder. However, in a monorepo setup like that found in a Yarn Workspace, the next package would be found in the workspace node modules.

To use this storybook-addon-next in Yarn Workspaces, I think there will need to be a way to specify where to find the next package.

One other option might be to look for the next package relative to where storybook-addon-next is installed (for a Yarn Workspace, this would be in the same node modules as next).

**Comment by RyanClementsHax** [comment]
Thanks for opening the discussion. Can you provide a minimal reproduction case? (I.e provide a repo that minimally replicates this feature or the lack thereof) I'm not familiar with yarn workspaces so I want to make sure I understand what you're talking about.

**Comment by RyanClementsHax** [comment]
see #23

**Reply by RyanClementsHax** [reply]
This should be supported with the changes made under #23. And the addition of the `nextConfigPath` option (see [options](https://github.com/RyanClementsHax/storybook-addon-next/pull/41)). Let me know if it ends up not working for you.
